owncloudData/src/test/java/com/owncloud/android/data/capabilities/datasources/OCLocalCapabilitiesDataSourceTest.kt

```diff
@@ -58,7 +59,7 @@ class OCLocalCapabilitiesDataSourceTest {
     }
 
     @Test
-    fun getCapabilitiesForAccountAsLiveData() {
+    fun `getCapabilitiesForAccountAsLiveData returns a livedata`() {
```

Comment:
I will add more descriptive names for the tests of this class.

Reply:
You didn´t add  more descriptive names for the rest of the tests.

Reply:
Done